src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ArrayOfEmbeddedPointersNode.cs

```diff
@@ -108,7 +108,7 @@ public EmbeddedPointerIndirectionWithSymbolNode(ArrayOfEmbeddedPointersNode<TTar
                 _id = id;
             }
 
-            public void AppendMangledName(NameMangler nameMangler, Utf8StringBuilder sb)
+            public void AppendMangledName(NameMangler nameMangler, Utf8StringBuilder sb, string compilationUnitPrefix)
```

Comment:
It would be better for the CompilationUnixPrefix to be a property on NameMangler instead of being passed everywhere.